src/Tracing/Span.php

```diff
@@ -567,6 +567,20 @@ public function toTraceparent(): string
         return sprintf('%s-%s%s', (string) $this->traceId, (string) $this->spanId, $sampled);
     }
 
+    /**
+     * Returns a string that can be used for the W3C `traceparent` header & meta tag.
+     */
+    public function toW3CTraceparent(): string
+    {
+        $sampled = '';
+
+        if ($this->sampled !== null) {
+            $sampled = $this->sampled ? '-01' : '-00';
+        }
+
+        return sprintf('00-%s-%s%s', (string) $this->traceId, (string) $this->spanId, $sampled);
```

Comment:
https://www.w3.org/TR/trace-context/#version-format

Hardcoding the version to 00 seems to be okay.